src/h3lib/include/h3api.h.in

```diff
@@ -251,6 +251,21 @@ double H3_EXPORT(degsToRads)(double degrees);
 double H3_EXPORT(radsToDegs)(double radians);
 /** @} */
 
+/** @defgroup pointDist pointDist
+ * Functions for pointDist
```

Comment:
We had previously not exposed these functions although we did have them. What is the reason they are exposed here?

Reply:
@nrabinowitz and I had discussed this elsewhere, but now I'm having trouble finding it.

Since the area methods depend on this function, and since there can be implementation differences in distance calculations (like the model of the earth one is using), exposing this to users allows them to check that some distance calculation they might be using matches our internal calculation, and could help identify where discrepancies are coming from in, say, area calculations.

This can also help users verify/understand the edge length functions, for example.

There are also users who want to calculate the distance between cell centroids (for example, in `h3-py`), and since we have this distance code internally anyway, exposing it allows a user to re-use it without having to write their own, and they can be sure that the distance calculations match our internal calculations.